common/core/mosaicdisplay.h

```diff
@@ -141,9 +150,20 @@ class MosaicDisplay : public NativeDisplay {
   bool pending_vsync_ = false;
   bool update_connected_displays_ = true;
 #ifdef ENABLE_PANORAMA
+  std::vector<NativeDisplay *> *virtual_panorama_displays_;
+  std::vector<NativeDisplay *> *physical_panorama_displays_;
+  std::vector<NativeDisplay *> real_physical_displays_;
+  DisplayManager *displaymgr_ptr_ = NULL;
+  SpinLock panorama_lock_;
   bool panorama_mode_ = false;
+  bool panorama_enabling_state_ = false;
+  bool skip_update_ = false;
```

Comment:
DRM displays are just part of child displays of mosaic display, so they are different in general. Anyway, since we have binder API for other process to control the HWC updating mechanism, we need to consider how to handle it properly for all display modes. We could address this in a separated PR. Thanks!